spot_wrapper/spot_dance.py

```diff
@@ -0,0 +1,83 @@
+import time
+
+from bosdyn.choreography.client.choreography import (
+    load_choreography_sequence_from_txt_file,
+)
+from bosdyn.client import ResponseError
+from bosdyn.client.exceptions import UnauthenticatedError
+from bosdyn.client.lease import LeaseKeepAlive
+from bosdyn.client.robot import Robot
+import typing
+
+
+class SpotDance:
+    def __init__(
+        self,
+        robot: Robot,
+        choreography_client: typing.Any,
```

Comment:
Can you put the actual [type](https://github.com/boston-dynamics/spot-sdk/blob/master/python/bosdyn-choreography-client/src/bosdyn/choreography/client/choreography.py) instead of Any